### PR TITLE
Fixes a very silly clockwork false wall trick

### LINKED ISSUE #27986: by making a clockwork wall into a falsewall and opening it, you can proselytise a second clockwork wall on the same spot as the first clockwork wall

### PR DESCRIPTION
**Pull request by ChangelingRain**
Fixes #27986

### CODE REVIEW

**Comment by loqoman** [comment]
Read: Review

File: code/game/gamemodes/clock_cult/clock_helpers/proselytizer_helpers.dm [hunk 1]
```diff
@@ -49,6 +49,9 @@
 /turf/open/floor/clockwork/proselytize_vals(mob/living/user, obj/item/clockwork/clockwork_proselytizer/proselytizer, silent)
 	if(locate(/obj/structure/table) in src)
 		return FALSE
+	if(locate(/obj/structure/falsewall) in contents)
+		to_chat(user, "<span class='warning'>There is a false wall in the way, preventing you from proselytizing [src] into a clockwork wall.</span>")
```
Comment (Cyberboss) [inline code/game/gamemodes/clock_cult/clock_helpers/proselytizer_helpers.dm]: \the src?

Comment (RemieRichards) [inline code/game/gamemodes/clock_cult/clock_helpers/proselytizer_helpers.dm]: @Cyberboss ```[src]``` automatically adds ```\the``` to it.

Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_helpers/proselytizer_helpers.dm]: I swear I told you this after you suggested using [src] for the gang dominator name...

Comment (RemieRichards) [inline code/game/gamemodes/clock_cult/clock_helpers/proselytizer_helpers.dm]: You did, we both did.

**Pull request merged**